api/db/db.js

```diff
@@ -0,0 +1,7 @@
+import mongoose from 'mongoose';
+
+mongoose.connect('mongodb+srv://huntinator_server:UYjTXTeo6xyxrGUV@huntinator-crnnx.mongodb.net/test?retryWrites=true&w=majority', { useNewUrlParser: true }, err => {
```

Comment:
Firstly, vvhat za hell iz zat URL?
2ndly, put it a const and pass it in just to make it look a bit better, and if it's a URL that could be split up into a couple of variables that would be even nicer just to make it a bit more readable.

Reply:
That's an `SRV` url which is a protocol that mongodb atlas uses to connect to a database hosted on there servers. It contains the username and password for the database account. I'll move it to a const above but it would be pointless to split it up as it's an autogenerated string.

Reply:
I added the password as an env var and added it to the heroku server - better that way

Reply:
Yeah, in that case, it, you're right it shouldn't be split up. It's a good thing you store that  as an env var now  👍